WordPress/Classes/ViewRelated/Reader/ReaderDiscoverHeaderView.swift

```diff
@@ -72,6 +82,19 @@ final class ReaderDiscoverHeaderView: UIView {
             view.isSelected = view.channel == selectedChannel
         }
     }
+
+    // MARK: UITextViewDelegate
+
+    func textView(_ textView: UITextView, shouldInteractWith URL: URL, in characterRange: NSRange) -> Bool {
```

Comment:
What do you think about adding some defensive code here to make sure only `/interest` URL is handled? Also, maybe add an assertion failure for other URLs.